sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryClientAsyncTest.java

```diff
@@ -5,92 +5,21 @@
-        TestUtils.importImage(ImplUtils.getTestMode(), HELLO_WORLD_REPOSITORY_NAME, Arrays.asList("latest", "v1", "v2", "v3", "v4"));
-        TestUtils.importImage(ImplUtils.getTestMode(), ALPINE_REPOSITORY_NAME, Arrays.asList("latest"));
+    private ContainerRegistryAsyncClient getContainerRegistryAsyncClient() {
+        return getContainerRegistryBuilder(new LocalHttpClient(recordFileName)).buildAsyncClient();
     }
 
-    @AfterAll
-    static void afterAll() {
-        StepVerifier.resetDefaultTimeout();
-    }
-
-    private ContainerRegistryAsyncClient getContainerRegistryAsyncClient(HttpClient httpClient) {
-        return getContainerRegistryBuilder(httpClient).buildAsyncClient();
-    }
-
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
-    @MethodSource("getHttpClients")
-    public void listRepositories(HttpClient httpClient) {
-        client = getContainerRegistryAsyncClient(httpClient);
-
-        StepVerifier.create(client.listRepositories())
-            .recordWith(ArrayList::new)
-            .thenConsumeWhile(x -> true)
-            .expectRecordedMatches(repositories -> repositories.containsAll(Arrays.asList(TestUtils.HELLO_WORLD_REPOSITORY_NAME, TestUtils.ALPINE_REPOSITORY_NAME)))
-            .verifyComplete();
-    }
-
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
-    @MethodSource("getHttpClients")
-    public void listRepositoriesWithPageSize(HttpClient httpClient) {
-        client = getContainerRegistryAsyncClient(httpClient);
-
-        StepVerifier.create(client.listRepositories().byPage(PAGESIZE_2))
-            .assertNext(res -> {
-                List<String> repositories = res.getValue();
-                assertTrue(repositories.size() <= PAGESIZE_2);
-
-                Stream<String> repoStream = repositories.stream();
-                assertTrue(repoStream.filter(a -> TestUtils.ALPINE_REPOSITORY_NAME.equals(a)).findAny().isPresent());
-                assertNotNull(res.getContinuationToken());
-            }).expectComplete();
-
-        // Not testing for complete since there are others as well.
-    }
-
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
-    @MethodSource("getHttpClients")
-    public void listRepositoriesWithInvalidPageSize(HttpClient httpClient) {
-        client = getContainerRegistryAsyncClient(httpClient);
-
-        StepVerifier.create(client.listRepositories().byPage(-1))
-            .expectErrorMatches(res -> res instanceof IllegalArgumentException)
-            .verify();
-    }
-
-    // What is the prescribed way of running delete test scenarios.
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
-    @MethodSource("getHttpClients")
-    public void deleteRepositoriesThrows(HttpClient httpClient) {
-        client = getContainerRegistryAsyncClient(httpClient);
+    @Test
+    public void deleteRepositoryThrows() {
+        recordFileName = "ContainerRegistryClientAsyncIntegrationTests.deleteRepositoryWithResponseThrows[1].json";
```

Comment:
Should just rename the recording file to match this tests name

Reply:
This recordFile is from another test run.